bin/validate_ssa.py

```diff
@@ -92,17 +104,16 @@ def test_detection(test, args):
     with open(test, 'r') as fh:
         test_desc = yaml.safe_load(fh)
         name = test_desc['name']
-        print("Testing %s" % name)
+        log(logging.INFO, "Testing %s" % name)
         # Download data to temporal folder
         data_dir = tempfile.TemporaryDirectory(prefix="data", dir=get_path("%s" % SSML_CWD))
         # Temporal solution
         if test_desc['attack_data'] is None or len(test_desc['attack_data']) == 0:
-            print("No dataset in testing file")
+            log(logging.ERROR, "No dataset in testing file in %s" % test)
             return False
         d = test_desc['attack_data'][0]
         test_data = os.path.abspath("%s/%s" % (data_dir.name, d['file_name']))
-        if args.debug:
-            print("Downloading dataset %s from %s" % (d['file_name'], d['data']))
+        log(logging.DEBUG, "Downloading dataset %s from %s" % (d['file_name'], d['data']))
```

Comment:
As above, `log(logging.DEBUG, 'Downloading dataset %s from %s', d['file_name'], d['data'])`, and likewise in a bunch of log statements below